cmake/SetupSpheral.cmake

```diff
@@ -77,7 +77,7 @@ if(ENABLE_OPENMP)
   list(APPEND SPHERAL_BLT_DEPENDS openmp)
 endif()
 
-if(ENABLE_CUDA)
+if(ENABLE_CUDA AND NOT SPHERAL_HIDE_GPU)
```

Comment:
Can you not set ENABLE_CUDA=off? is it because you're using Spheral as a submodule. Do we need to specialize this to something like SPHERAL_ENABLE_CUDA so you aren't having to throw SPHERAL_HIDE_GPU.

Reply:
I have changes that address this in one of the upcoming GPU PRs. Leave this how it is and we can discuss outside of this PR when it comes up again.

Reply:
Ok, I'll leave this commented out here for now.